index.html

```diff
@@ -25,7 +25,7 @@ <h1 class="pull-left">Find It!</h1>
           <label>Account</label>
           <span id="as-login-status" class="label label-warning"></span>
           <br/>
-          <button type="button" class="btn btn-warning button-sm" data-toggle="modal" data-target="#help">Help</button>
+          <button type="button" class="btn btn-warning button-sm" data-toggle="modal" data-target="#help" onclick="ga('send', 'event', 'search', 'click', 'button');">Help</button>
```

Comment:
Yeah, I think I just fundamentally don't understand getting google analytics to work and how to structure it. But I'll try again.